backend/src/main/java/com/woowacourse/momo/service/schedule/dto/SchedulesResponse.java

```diff
@@ -0,0 +1,36 @@
+package com.woowacourse.momo.service.schedule.dto;
+
+import com.woowacourse.momo.domain.schedule.Schedule;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public record SchedulesResponse(List<AttendeesScheduleResponse> schedules) {
+
+    public static SchedulesResponse from(List<Schedule> schedules) {
+        Map<LocalDateTime, List<String>> attendeesOfSchedules = groupingAttendeeByMeetingDateTime(schedules);
+        List<AttendeesScheduleResponse> scheduleResponses = convertToAttendeesResponses(attendeesOfSchedules);
+        return new SchedulesResponse(scheduleResponses);
+    }
+
+    private static Map<LocalDateTime, List<String>> groupingAttendeeByMeetingDateTime(List<Schedule> schedules) {
+        return schedules.stream()
+                .collect(Collectors.groupingBy(
+                        Schedule::getDateTime,
+                        Collectors.mapping(schedule -> schedule.getAttendee().name(), Collectors.toList())
+                ));
+    }
+
+    private static List<AttendeesScheduleResponse> convertToAttendeesResponses(
+            Map<LocalDateTime, List<String>> attendeesOfSchedules
+    ) {
+        return attendeesOfSchedules.entrySet().stream()
+                .map(attendeesOfSchedule -> new AttendeesScheduleResponse(
+                        attendeesOfSchedule.getKey().toLocalDate(),
+                        attendeesOfSchedule.getKey().toLocalTime(),
+                        attendeesOfSchedule.getValue()
+                ))
+                .toList();
```

Comment:
`keySet()`으로 디미터의 법칙을 지킬 수 있을 것 같은데 어떠실까요?
```suggestion
        return attendeesOfSchedules.keySet().stream()
                .map(dateTime -> new AttendeesScheduleResponse(
                        dateTime.toLocalDate(),
                        dateTime.toLocalTime(),
                        attendeesOfSchedules.get(dateTime)
                ))
                .toList();
```

Reply:
그렇네요! 적극 반영하겠습니다~~!!